src/pages/minsuKang/Login/LoginMinsu.js

```diff
@@ -0,0 +1,43 @@
+import React, {useState} from "react";
+import "./Login.scss";
+import {useNavigate} from "react-router-dom";
+
+
+
+
+function Login(){
+    const navigate = useNavigate();
+    const [userId, setUserId] = useState("")
+    const [userPw, setUserPw] = useState("") //이처럼 usestate를 통해 state를 정해줘야만 값이 저장된다. 
+    const [active, setActive] = useState(false)
+    const handleIdInput = (e)=>{ setUserId(e.target.value);}// 여기서 잘 못 생각한 부분: userid라는 것이 결국에는 검출되는 것이므로. 함수과정을 거치면서 변화된다는 생각을 못함.
+    const handlePwInput = (e)=>{ setUserPw(e.target.value);}
+    const goToList = () => {if (userId.includes('@') && userPw.length >5) 
+    {sessionStorage.setItem("id",userId); navigate('/list-minsu');} 
+    else { alert('가입된 회원이 아닙니다. 회원가입을 먼저 해주세요.');}}
+    const activeLogin = ()=> { return userId.includes('@') && userPw.length>5 ? setActive(true) : setActive(false);}
+    
+return(
+    <div className="Login_container">
+        <div className="Login_wrapper">
+                <p className="Login_logo">WeBucks</p>
+                <form  action="submit" className="Login_button1">
+                    <input  id= "id" className="Login_id" type="text" placeholder="전화번호, 사용자 이름 또는 이메일" onKeyUp= {activeLogin} onChange={handleIdInput} />
+                    <input  id ="pw" className="Login_pw" type="password" placeholder="비밀번호"onKeyUp= {activeLogin} onChange={handlePwInput}/>
```

Comment:
```suggestion
                    <input  id= "id" className="Login_id" type="text" placeholder="전화번호, 사용자 이름 또는 이메일" onChange={handleIdInput} />
                    <input  id ="pw" className="Login_pw" type="password" placeholder="비밀번호"onKeyUp= {activeLogin} onChange={handlePwInput}/>
```

하나의 요소에 비슷한 이벤트를 두 개 사용할 필요가 없을 것 같습니다. activeLogin 함수의 내용을 handlePwInput 안으로 옮겨 한 번에 처리하는 것이 좋을 것 같습니다!

Reply:
아 네 그렇게 한 번에 바꿔도 되겠군요~ 한 번 고려해서 수정해 보겠습니다